backend/src/common/duckdb/README.md

```
@@ -0,0 +1,308 @@
+# DuckDB Sidecar Module
+
+Módulo NestJS para executar DuckDB em um processo separado (sidecar), com suporte a:
+- **Vector Similarity Search** (HNSW index) - para busca semântica com embeddings CLIP
+- **Full-Text Search** (BM25) - para busca textual
+- **Hybrid Search** - combinação de ambas as abordagens
+- **PostgreSQL integration** - attach direto ao banco Postgres
+
+## Arquitetura
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)